### PR TITLE
Sy5

### PR DESCRIPTION
**Pull request by efjacobson**
@gdbrown i am going to go over this again from top down but everything is compiling ok and all tests are passing

### CODE REVIEW

**Comment by gdbrown** [comment]
@efjacobson let's also do the fgets in php insertion points.  (fget is not on js btw).  we're also going to do all the concretes in triniti and mark the old mixins as deprecated (all the command/event/request/response that are mixins).

File: build/js/tests/bootstrap.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 require('@babel/register')({
-  ignore: [/node_modules\/(?!@gdbots|lodash-es)/],
+  ignore: [/node_modules\/(?!@gdbots|@triniti|lodash-es)/],
```
Comment (efjacobson) [inline build/js/tests/bootstrap.js:2]: @gdbrown does this make sense? we are in triniti

Comment (gdbrown) [reply build/js/tests/bootstrap.js:2]: @efjacobson you're right, it makes no sense here

**Comment by efjacobson** [comment]
@gdbrown all command/event/request mixins have been deprecated and remade as concrete

**Comment by gdbrown** [comment]
@efjacobson let's also make search request/responses for all the list-all-blah and get-all-blah request/response.  we index all things now so we can use searching for those.  e.g.

- https://github.com/gdbots/schemas/blob/master/schemas/gdbots/iam/request/search-roles-request/latest.xml
- https://github.com/gdbots/schemas/blob/master/schemas/gdbots/iam/request/search-roles-response/latest.xml

vs

- https://github.com/gdbots/schemas/blob/master/schemas/gdbots/iam/mixin/list-all-roles-request/latest.xml
- https://github.com/gdbots/schemas/blob/master/schemas/gdbots/iam/mixin/list-all-roles-response/latest.xml

no concrete list-all-blah or get-all-blah

**Comment by efjacobson** [comment]
@gdbrown do you want sort field and enums for the new search requests?

**Comment by gdbrown** [comment]
@efjacobson yes, ref https://github.com/gdbots/schemas/blob/master/schemas/gdbots/iam/enums.xml#L7

**Comment by efjacobson** [comment]
didnt include relevance/popularity as they dont seem, well, relevant (or popular)

**Review by gdbrown — APPROVED** [review]

**Pull request merged**